Tasks/AzureAppServiceManage/task.json

```diff
@@ -43,7 +43,10 @@
                 "Start Azure App Service": "Start App Service",
                 "Stop Azure App Service": "Stop App Service",
                 "Restart Azure App Service": "Restart App Service",
-                "Install Extensions": "Install Extensions"
+                "Install Extensions": "Install Extensions",
+                "Enable Continuous Monitoring": "Enable Continuous Monitoring",
+                "Start Webjobs": "Start Webjobs",
```

Comment:
Can we make it explicit - "Start all continuous webjobs" and "Stop all continuous webjobs"?